std/typecons.d

```diff
@@ -2853,6 +2874,16 @@ Returns:
     assert(app.data == "1");
 }
 
+// Issue 19799
+@safe unittest
+{
+    import std.format : format;
+
+    const Nullable!string a = const(Nullable!string)();
+
+    format!"%s"(a);
```

Comment:
How about asserting on the output here?

Reply:
I don't think this code wants to care what the output is, as long as there is one. I think that'd be misleading considering it's not the point of the bug, and hence not the point of the unittest.

Reply:
Assertions added, purely for coverage's sake.